python/mxnet/gluon/estimator/estimator.py

```diff
@@ -125,16 +113,18 @@ def __init__(self, net,
             if not self._is_initialized():
                 self.net.initialize(ctx=self.context)
 
-        # handle trainers
-        if isinstance(trainers, gluon.Trainer):
-            self.trainers = [trainers]
-        else:
-            self.trainers = trainers or []
-        if not self.trainers:
+        # handle trainer
+        if not trainer:
             warnings.warn("No trainer specified, default SGD optimizer "
                           "with learning rate 0.001 is used.")
-            self.trainers = [gluon.Trainer(self.net.collect_params(),
-                                           'sgd', {'learning_rate': 0.001})]
+            self.trainer = gluon.Trainer(self.net.collect_params(),
+                                           'sgd', {'learning_rate': 0.001})
+        elif not isinstance(trainer, gluon.Trainer):
+            raise ValueError("Trainer must be a Gluon Trainer instance, refer to gluon.trainer")
```

Comment:
Since you say refer to gluon.trainer, you could probably also add a url to gluon.trainer docs : http://mxnet.incubator.apache.org/versions/master/api/python/gluon/gluon.html#trainer